app/views/shared/_custom_styles.html.erb

```diff
@@ -0,0 +1,171 @@
+<style>
```

Comment:
This seems like it's worthy of its own css file with this amount of styling. Is there a reason this is in an html.erb file instead of a css.erb file? It should still be able to access all of the same ruby functionality here without being separated from the rest of the styling in the file structure.